emote/extra/schedules.py

```diff
@@ -25,10 +25,11 @@ def __init__(self, initial: float, final: float, steps: int):
         self.steps = steps
 
         self._step_count = 0
-        self._last_val = initial
+        self._val = initial
```

Comment:
Hm. I don't actually like _val. Could we maybe go for something a bit more descriptive here? Since we call it `value_min` and `value_max` I think just `current_value` would be nicer.